openlibrary/templates/books/custom_carousel.html

```diff
@@ -71,7 +71,7 @@ <h2 class="home-h2"><a href="$url">$title</a></h2>
         </div>
         <script type="text/javascript">
               window.q.push(function() {
-                Carousel.add('.carousel-$key', 6, 5, 4, 3, 2, 1);
+                Carousel.add('.carousel-$key', 6, 5, 4, 3, 2, 1, '$key', '$load_more_url');
```

Comment:
QUESTION: You sure that `$key` and `$load_more_url` will never have single quotes in them?